### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by Xeko-92** [sunflower-land/sunflower-land#3932] [General]
-

**Comment by Dionis404** [comment]
Only moderators can issue chat restrictions in discord.

**Reply by Dionis404** [reply]
And yes, you are wrong, the faction bonus is given to all faction members. 
Where you think there will be 0, 1 is used.

**Reply by Xeko-92** [reply]
> Only moderators can issue chat restrictions in discord.

Well then moderators are playing favourites because he was just as "toxic" as I was.

**Reply by Dionis404** [reply]
You said "gtfo" which is an insult.

**Reply by Xeko-92** [reply]
Calling someone silly is an insult, depending on the context. Saying get the fuck out, isn't an insult. It can be considered inappropriate language, but I'm not insulting him.

**Reply by Xeko-92** [reply]
Before that he also called my behaviour toxic, which is also an insult.

**Reply by Dionis404** [reply]
I don't want to argue with you. 
Just know that you're wrong, and experience is for everyone.

**Reply by Xeko-92** [reply]
Lol alright. I'm over dealing with this biased community anyways. Good luck ✌️

**Reply by eliasSFL** [reply]
@Xeko-92 For full transparency, I was the one who muted you, squarely because you insulted spanks and tell him to f off. But I now recognise that spanks was also in the wrong for calling you toxic and all as well. I will talk to Spanks about his behaviour and I have removed your timeout on discord. But please be gracious to other community members, don't call people names and insult them

**Reply by Xeko-92** [reply]
Thank you for your transparency. I wish you had said something in chat at the time and told us both to stop rather than just muting me. It felt one-sided because I didn't think I was any more in the wrong than he was.

**Comment by Jericho-rpc** [comment]
It's a non-democratized community, they will kick you out of the community for some of your comments and suggestions, I know of many players around me who have been kicked out of Discord for voicing some of their opinions

**Comment by adamhannigan** [comment]
Thanks for raising the discussion @Xeko-92 

@Jericho-rpc  To date, no one has been kicked out for voicing opinions. We encourage this in all forms. We do have strict rules around harassment, scams, misinformation etc. though.

I can't speak directly to this case but it likely would of breached the ToS of the Discord. We will be able to resolve this 👍
